include/z64actor.h

```diff
@@ -571,11 +571,12 @@ typedef enum NaviEnemy {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
-typedef struct TargetContextEntry {
+// A set of 4 triangles which appear as a ring around an actor when the player Z-Targets it
+typedef struct {
```

Comment:
add struct name?

Reply:
yeah this was copy pasted from a pr that didnt have the struct name fix, will do